### PR TITLE
feat(linear-gradient-&-some-fixes): linear gradient in start screen a…

### PR DESCRIPTION
**Pull request by guardian-001**
## What does this do?

This PR includes several changes to improve the application's code quality and responsiveness:

_layout component: Fixed the component name.
Login header: Removed unnecessary spaces.
Login form: Removed unnecessary spaces.
Conditions: Fixed spaces between texts.
Login button: Made text responsive.
Main button: Made text responsive.
Start screen: Added a linear gradient and fixed the Stack.Screen.
Checkbox shared component: Made text responsive.
Input shared component: Made text responsive.
New library installation: Installed a new library to implement the linear gradient.

## Why did you do this?

The primary motivation behind these changes is to:

Enhance code readability and maintainability by removing unnecessary spaces.
Improve the user experience by making text elements responsive, ensuring the app looks good on tablets and phones.
Implement a linear gradient to enhance the visual appeal of the start screen.

## Who/what does this impact?

These changes impact:

Developers: Simplifies the codebase, making it easier to read and maintain.
User Experience: Improves the appearance and usability of the app on various devices, including tablets and phones.
Visual Consistency: Adds a linear gradient for a more modern and visually appealing start screen.

## How did you test this?

Manual Testing: Tested on phones and tablets using an Android emulator to ensure responsiveness and visual consistency.
Unit Testing: Conducted unit tests to verify that the changes did not introduce any regressions

image
image

### CODE REVIEW

File: src/modules/shared/conditions.tsx [hunk 1]
```diff
@@ -15,17 +15,19 @@ export default function Conditions() {
           onPress={() => {
             Linking.openURL(process.env.TERMS_URL || '');
           }}
-          className={`text-xs font-bold text-primary-txt  `}
+          className={`text-xs font-bold text-primary-txt`}
         >
+          {' '}
```
Comment (MohamedMegdich1) [inline src/modules/shared/conditions.tsx:20]: Please remove the empty space in the code.

File: src/modules/visitor/start-screen.tsx [hunk 2]
```diff
@@ -12,46 +13,42 @@ import { Container, MainButton } from '../shared';
...
-        }}
-      />
-
-      <Container style="flex-1 items-center justify-center w-full my-4  pt-5">
-        <Container style="flex   items-start justify-center w-[86%]   ">
-          <ImageContainer className="flex-start mb-5   flex h-10 w-10 justify-center">
+    <LinearGradient
+      start={{ x: 0, y: 1 }}
+      end={{ x: 0, y: 0 }}
+      colors={['#E5EBFF', '#DFF6FF']}
```
Comment (MohamedMegdich1) [inline src/modules/visitor/start-screen.tsx:22]: Avoid using static colors in the code.

**Review by MohamedMegdich1 — CHANGES_REQUESTED** [review]

File: src/modules/shared/conditions.tsx (hunk 1 above)
Comment (guardian-001) [reply src/modules/shared/conditions.tsx:20]: I have no solution to add spaces dynamically for translated text. The only two solutions are to add white space or add margin between the texts. Which do you prefer?

File: src/modules/visitor/start-screen.tsx (hunk 2 above)
Comment (guardian-001) [reply src/modules/visitor/start-screen.tsx:22]: image
image
fixed

File: src/modules/shared/conditions.tsx (hunk 1 above)
Comment (guardian-001) [reply src/modules/shared/conditions.tsx]: fixed

**Review by MohamedMegdich1 — APPROVED** [review]

**Pull request merged**